doc/releases/release-notes-3.3.rst

```diff
@@ -269,6 +269,22 @@ Bluetooth
 
 * Audio
 
+  * Improve stream coupling for CIS as the unicast client
```

Comment:
Please use past tense.

```suggestion
  * Improved stream coupling for CIS as the unicast client
```

Reply:
Oh, thought we just copied the commit oneliners here :) But sure, I can modify it

Reply:
At some point, we seem to have agreed on using the past tense in the release notes (not a bad decision since it is a quite common practice), and [we now have it as part of the release checklist](https://github.com/zephyrproject-rtos/zephyr/issues/54495) :)

Please update the rest as well.

Reply:
How do you suggest I make 
```
  * Vol rend use `bt_vcp_vol_rend` instead of `bt_vcp`
  * Use `bt_vcp_vol_ctlr` instead of `bt_vcp`
```

past tense? Or should I just rewrite the entire line?

Reply:
something like:

* Updated vol rend to use `bt_vcp_vol_rend` instead of `bt_vcp`
* Deprecated `bt_vcp` in favour of `bt_vcp_vol_ctlr`.

Reply:
The CI check doesn't stop those though ;) But will update, thanks